go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/elastic/go-docappender
 go 1.21
 
 require (
+	github.com/elastic/go-elasticsearch/v7 v7.17.10
```

Comment:
WDYT about not having this dependency? I think if we separate the testing into a separate `go.mod` then we can do without this.